server/resolvers/TestPlanReport/issuesResolver.js

```diff
@@ -30,8 +30,13 @@ const issuesResolver = async testPlanReport => {
                 html_url,
```

Comment:
This is for the case of general feedback, where the title does not include `[Test #]`

Reply:
Good catch!